draft-ietf-suit-information-model.md

```diff
@@ -799,18 +803,25 @@ If key expiration is performed based on time, then a secure clock is needed. If
 
 Mitigates: [THREAT.KEY.EXPOSURE](#threat-key-exposure)
 
+Implemented by: Secure storage technology, which is a system design/implementation aspect outside the scope of the manifest format. 
+
+
 ### REQ.SEC.MFST.CHECK: Validate manifests prior to deployment {#req-sec-mfst-check}
 
 Manifests SHOULD be verified prior to deployment. This reduces problems that may arise with devices installing firmware images that damage devices unintentionally.
 
 Mitigates: [THREAT.MFST.MODIFICATION](#threat-mfst-modification)
 
+Implemented by: Testing infrastructure. While outside the scope of the manifest format, proper testing of low-level software is essential for avoiding unnecessary down-time or worse situations. 
```

Comment:
I think we should add a little more text here. The intent here, though it's not clear, is that an update **distribution** system should pre-validate manifests before distribution. This is in addition to normal test infrastructure.